stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/header/MessagesHeaderView.kt

```diff
@@ -142,6 +161,20 @@ public class MessagesHeaderView : ConstraintLayout {
         binding.avatar.isVisible = false
     }
 
+    public fun setNormalMode() {
+        binding.title.text = normalModeTitle
+        binding.onlineLabel.text = normalModeSubTitle
+    }
+
+    public fun setThreadMode(isGroupThread: Boolean) {
+        val title = threadModeTitle ?: normalModeTitle
```

Comment:
If `threadModeTitle` ever ends up being `null` (i.e. the attribute was not set), do we really want to show `normalModeTitle` twice on the header? Perhaps we could use our own `"Thread Reply"` string by default for the main title, we have a string resource for it already anyway.

Reply:
"Thread Reply" only makes sense for someone that speaks english... I think that simply not changing the title (using normalModeTitle) would be better

Reply:
All of our UI ships with English text on it by default, I don't see how this is a problem with this specific header text